pkg/controllers/disruption/controller.go

```diff
@@ -114,6 +114,9 @@ func (c *Controller) Reconcile(ctx context.Context, _ reconcile.Request) (reconc
 	// with making any scheduling decision off of our state nodes. Otherwise, we have the potential to make
 	// a scheduling decision based on a smaller subset of nodes in our cluster state than actually exist.
 	if !c.cluster.Synced(ctx) {
+		if time.Since(c.cluster.LastSyncTime()) > state.AbnormalClusterSyncDuration { 
+			logging.FromContext(ctx).Infof("cluster state has not been in sync since %v, waiting for sync", c.cluster.LastSyncTime()) 
+		}
 		logging.FromContext(ctx).Debugf("waiting on cluster sync")
```

Comment:
Should we just remove this now, or do we want to keep it and put a much lower verbosity when the controller runtime logging change goes in?

Reply:
We should just be able to remove it.

Reply:
removed